espanol/dolor/dolor.md

```diff
@@ -1,2 +1,2 @@
 Y todo era oscuro. Y la luz no volvia. Y el cielo estaba abajo. Y nada era lo que debia.
-Dolor.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.